src/tests/integration/users-query.test.ts

```diff
@@ -0,0 +1,226 @@
+import { UserModel } from '../../data-access/models/user';
+import { createTestClient } from 'apollo-server-testing';
+import { server } from '../../lambda';
+import { awsResponse, getFn, queryFn, scanFn } from '../../__mocks__/aws-sdk/clients/dynamodb';
+import { accessTokenEnvName } from '../../dist/configuration/mapbox';
+import { nameIndexEnvName, usersTableEnvName } from '../../configuration/users-db';
+import { WithIndexSignature } from '../../utils/types';
+import { BatchGetResponseMap, ItemList } from 'aws-sdk/clients/dynamodb';
+import { Maybe } from 'graphql/jsutils/Maybe';
+import moment from 'moment';
+
+describe('Query by id', () => {
+	const mockUser: UserModel = {
+		id: '1234',
+		address: 'Brasilia, Brazil',
+		dob: '1995-09-03T00:00:00.000Z',
+		name: 'Test user',
+		description: 'Test description',
+		createdAt: '2021-05-01T17:52:48.299Z',
+		updatedAt: '2021-06-01T17:52:48.299Z',
+	};
+
+	const setup = () => {
+		process.env[usersTableEnvName] = 'test-table';
+		process.env[accessTokenEnvName] = 'test-access-token';
+		return createTestClient(server);
+	};
+
+	it('Should return all fields if found', async () => {
+		const { query } = setup();
+
+		awsResponse.mockReturnValueOnce(Promise.resolve({ Item: mockUser }));
+		const result = await query({
+			query: `
+                query {
+                    user(id: ${mockUser.id}) {
+                        id
+                        name
+                        address
+                        dob
+                        description
+                        createdAt
+                        updatedAt
+                        imageUrl
+                    }
+                }
+            `,
```

Comment:
Just curious why you chose to indent the string with spaces and code with tabs

Reply:
This was a mistake on my part. When I was developing these integration tests I must have copied the query text from Insomnia, my bad on this one. I should fix this.